database/src/main/kotlin/no/nav/su/se/bakover/database/revurdering/GjenopptakAvYtelsePostgresRepo.kt

```diff
@@ -93,6 +95,11 @@ internal class GjenopptakAvYtelsePostgresRepo(
                         vilkår = revurdering.vilkårsvurderinger.formue,
                         tx = tx,
                     )
+                    utlandsoppholdVilkårsvurderingRepo.lagre(
```

Comment:
Burde kanskje lage et moder-repo for vilkår som tar i mot vilkårsvurderinger eller kanskje både grunnlag og vilkårsvurderinger? Så kan vi evt. på sikt åpne for de spesifikke operasjonene dersom vi trenger det i fremtiden.

Reply:
Det bør vi nok vurdere etterhvert. Kunne også sett på effektivisering av lagringen i den sammenheng.. ender vel fort opp med at lagring av 1 vilkår fører til at vi lagrer alle andre vilkår og grunnlag.